package.json

```diff
@@ -28,12 +28,12 @@
     "build": "npm run preprocess && npm run bundle && npm run header && npm run stats",
     "cibuild": "npm run preprocess && node tasks/cibundle.js",
     "watch": "node tasks/watch.js",
-    "lint": "eslint --version && eslint . || true",
-    "lint-fix": "eslint . --fix",
+    "lint": "eslint --version && eslint .",
+    "lint-fix": "eslint . --fix || true",
     "docker": "node tasks/docker.js",
     "pretest": "node tasks/pretest.js",
     "test-jasmine": "karma start test/jasmine/karma.conf.js",
-    "citest-jasmine": "karma start test/jasmine/karma.ciconf.js",
+    "citest-jasmine": "CIRCLECI=0 karma start test/jasmine/karma.conf.js",
```

Comment:
Oh right. Thanks for checking!

`npm run citest-jasmine` is now effectively obsolete. But the CI and local case can now be handled via `npm run test-jasmine`. 
I kept `"citest-jasmine"` to not perturb workflows for other devs. But if I get enough :+1: I will gladly :hocho: it. 

Moreover, this should really be:

```
"citest-jasmine": "CIRCLECI=1 karma start test/jasmine/karma.conf.js",
```

but funny story `!!"0"` in JS evaluates to `true`, not to be confused with `!!0` which is `false`.